docs/operators_and_staking/operators.md

```diff
@@ -320,5 +327,4 @@ Only the account who registered **Operator** can swith the domain.
 
 :::note
 Stake of your **Nominators** won't be released, but will be moved to the new domain as well. 
-:::
-
+:::
```

Comment:
Was adding this intended?

Reply:
It's just a formatting highlight, I suppose. No real changes here